src/main/java/org/broadinstitute/hellbender/tools/exome/SegmentUtils.java

```diff
@@ -213,6 +217,30 @@ public static SimpleInterval trimInterval(final Locatable interval,
                 dataLine -> toModeledSegment(dataLine, false));
     }
 
+    /** Read only the sample names from a segment file.
+     *
+     * @param segmentFile Never {@code null}
+     * @return a list of the unique sample names in the seg file.
+     */
+    public static List<String> readSampleNamesFromSegmentFile(final File segmentFile) {
```

Comment:
I cannot use readSegmentFile, because I have to output a ``List<T extends Locatable>`` , which String is not.

I could create a more generic readSegmentFile, but I would rather punt until we refactor.  Is that okay?